src/lib.rs

```diff
@@ -265,10 +265,10 @@ impl<K: EnrKey> Enr<K> {
         self.seq
     }
 
-    /// Reads a custom key from the record if it exists, decoded as data.
+    /// Reads a custom key from the record if it exists, decoded as data. Caution! Only use for
+    /// data that is not an aggregate type. Default to using [`get_decodable`](Enr::get_decodable).
```

Comment:
This is fair. 

Potentially we might remove this function in the future. Apart from backwards compatibility, I would think that `get_decodable` should handle every case. 

Even in cases where the object is not RLP encodable, the default value would be bytes.

Reply:
I thought the same thing, that this can stay for backwards compatibility. what about adding
```suggestion
    /// data that is not an aggregate type. Default to using [`get_decodable`](Enr::get_decodable).
    #[deprecated(since = "0.11.2", note = "use get_decodable")]
```